src/components/shared/MarkerTooltipContents.js

```diff
@@ -80,12 +80,13 @@ function getMarkerDetails(data: MarkerPayload): React.Element<any> | null {
                         formatBytes
                       )
                     )}
-                  {_markerDetail(
-                    'gcnewnurserysize',
-                    'New nursery size',
-                    nursery.new_capacity,
-                    formatBytes
-                  )}
+                  {nursery.new_capacity &&
```

Comment:
The other problem with using the short circuit is especially visible in React: if the value is "0" (it's a number, so it's theorically possible), we won't access _markerDetail but instead just output the string "0".

Better explicitely check for "undefined"...

Likely the same issue with `lazy_capacity` below.

Reply:
I understand that "0" isn't possible from the C++ code -- but 1. we don't know it from the type, and 2. older profiles might have `0`.